lib/donations/currencies.rb

```diff
@@ -0,0 +1,47 @@
+# == Currency Converter to JSON
+#
+# Takes an array of integer amounts, in US cents, and
+# converts to GBP and EUR.
+#
+# ==== Examples
+#
+#   currencies = Donations::Currencies.for(102, 204, 302, 401)
+#   currencies.to_hash # also to_json
+#   =>
+#   {
+#     USD: [1.02, 2.04, 3.02, 4.01],
```

Comment:
My plan was to round to the closest integer under 20 and the nearest 5 for 25 and above